en.html

```diff
@@ -0,0 +1,472 @@
+									<img src="img/partners/honeywell.png" class="partner"/>
+								</a>
+							</div>
+						</div>
+
+					<div class="col-md-4 xsm-padding">
+						<div class="about">
+							<a target="_tab" href="https://www.redhat.com/en/global/czech-republic">
+								<img src="img/partners/redhat.png" class="partner"/>
+							</a>
+						</div>
+					</div>
+
+					<div class="col-md-4 xsm-padding">
+						<div class="about">
+							<a target="_tab" href="https://www.gatema.cz/">
+								<img src="img/partners/gatema.svg" class="partner"/>
+							</a>
+						</div>
+					</div>
+				</div>
+
+				<div class="row row-eq-height">
+					<div class="col-md-4 xsm-padding">
+						<div class="about">
+							<a target="_tab" href="https://www.fablabbrno.cz/">
+								<img src="img/partners/fablab.png" class="partner"/>
+							</a>
+						</div>
+					</div>
+
+					<div class="col-md-4 xsm-padding">
+						<div class="about">
+							<a target="_tab" href="https://www.soselectronic.cz/">
+								<img src="img/partners/sos.png" class="partner"/>
+							</a>
+						</div>
+					</div>
+
+					<div class="col-md-4 xsm-padding">
+						<div class="about">
+							<a target="_tab" href="https://laskarduino.cz/">
+								<img src="img/partners/laskarduino.png" class="partner"/>
+							</a>
+						</div>
+					</div>
+				</div>
+			</div>
+		</div>
+	</div>
+
+	<div id="gallery" class="section md-padding">
+		<div class="container">
+			<div class="row">
+				<div class="section-header">
+					<h2 class="title">Gallery</h2>
+				</div>
+
+				<div class="col-md-4 col-xs-6 work">
+					<a class="lightbox" href="./img/photo_1.jpg">
+						<img class="img-responsive" src="preview/photo_1.jpg" alt="">
+					</a>
+				</div>
+
+				<div class="col-md-4 col-xs-6 work">
+					<a class="lightbox" href="./img/photo_2.jpg">
+						<img class="img-responsive" src="preview/photo_2.jpg" alt="">
+					</a>
+				</div>
+
+				<div class="col-md-4 col-xs-6 work">
+					<a class="lightbox" href="./img/photo_3.jpg">
+						<img class="img-responsive" src="preview/photo_3.jpg" alt="">
+					</a>
+				</div>
+
+				<div class="col-md-4 col-xs-6 work">
+					<a class="lightbox" href="./img/photo_4.jpg">
+						<img class="img-responsive" src="preview/photo_4.jpg" alt="">
+					</a>
+				</div>
+
+				<div class="col-md-4 col-xs-6 work">
+					<a class="lightbox" href="./img/photo_5.jpg">
+						<img class="img-responsive" src="preview/photo_5.jpg" alt="">
+					</a>
+				</div>
+
+				<div class="col-md-4 col-xs-6 work">
+					<a class="lightbox" href="./img/photo_6.jpg">
+						<img class="img-responsive" src="preview/photo_6.jpg" alt="">
+					</a>
+				</div>
+			</div>
+		</div>
+	</div>
+
+	<div id="guide" class="section md-padding bg-grey">
+		<div class="container">
+			<div class="row">
+				<div class="section-header">
+					<h2 class="title">Production guide</h2>
+					<p>We prepared the whole <a href="guide.html">production guide</a>. 						
```

Comment:
We prepared -> Assembly instructions can be found ...
production -> assembly (viz předchozí)

Reply:
Assembly instructions can be found on the guide page.